sdk-core/transactions-factories/relayed_transactions_factory.md

```diff
@@ -0,0 +1,19 @@
+## RelayedTransactionsFactory
+
+class RelayedTransactionsFactory:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+    // Generally speaking, the constructor should be parametrized with a configuration object which defines entries such as:
+    // "minGasLimit", "gasLimitPerByte", gas limit for specific operations etc.
+
+    create_relayed_v1_transaction({
+        inner_transaction: ITransaction;
+        relayer_address: IAddress;
+        relayer_nonce: Optional[int];
+    }): Transaction;
+
+    // can throw InvalidGasLimitForInnerTransaction
+    create_relayed_v2_transaction({
+        inner_transaction: ITransaction;
```

Comment:
in case of inner transaction for v2, gas limit should also be specified, since it will be set in the outer transaction that relays the inner one. inner transaction's gas limit must be 0

Reply:
Indeed. Fixed